packages/testing/src/api/index.ts

```diff
@@ -168,8 +171,16 @@ export function loadDatabaseRules(
         body: options.rules
       },
       (err, resp, body) => {
+        let parsedBody: LoadDatabaseRulesErrorBody
+        try {
+          parsedBody = JSON.parse(body)
```

Comment:
I would consider the case where the body is invalid JSON to be an error (and an exceptional one, the emulator should always return valid JSON). Can we drop this `try` and blow up if that happens?

Reply:
Does the emulator return a body if the methods succeeds? We don't want the method to explode on a valid response.
If you don't know then I'll test further and implement accordingly on tuesday.

Reply:
It does, the body will look like `{"status":"ok"}`

Reply:
okay! will change on tuesday.